docs/essentials/calendar.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Xamarin.Essentials: Calendar"
+description: "This document describes the Calendar class in Xamarin.Essentials, which lets you view the user's calendar and event information."
+ms.assetid: a6d5f2a2-0f49-4726-9a74-8e7667ddb127
+author: ScottBTR
+ms.author: ScottYoung-5180
+ms.date: 03/12/2019
+---
+
+# Xamarin.Essentials: Calendar
+
+The **Calendar** class lets you view the user's calendar and event information.
+
+## Get started
+
+[!include[](~/essentials/includes/get-started.md)]
+
+To access the **Calendar** functionality the following platform specific setup is required.
+
+# [Android](#tab/android)
+
+The `Read Calendar` and `Write Calendar` permission is required and must be configured in the Android project. 
+
+Open the **AndroidManifest.xml** file under the **Properties** folder and add the following inside of the **manifest** node.
+
+```xml
+<uses-permission android:name="android.permission.READ_CALENDAR" />
+<uses-permission android:name="android.permission.WRITE_CALENDAR" />
```

Comment:
Perhaps I should be removing WRITE_CALENDAR at this stage?

Reply:
yep I think so

Reply:
Done